### PR TITLE
fix: 梳理单元格对齐方式, close #1084

### LINKED ISSUE #1084: 🤔 角头/行头/列头/单元格 之间的textAlign是会有冲突的么
### 🖋 Description
 比如设置了单元格的textAlign是left, 列头的textAlign是right, 此时只有列表头的最后一行是跟随单元格的设置, 并且列表头的textAlign不生效,  角头的设置也是如此,   想要的效果就是这几个的textAlign可以单独设置.

### PR DESCRIPTION
**Pull request by wjgogogo**
### 👀 PR includes

✨ Feature

- [ ] New feature

🎨 Enhance

- [ ] Code style optimization
- [ ] Refactoring
- [ ] Change the UI
- [ ] Improve the performance
- [ ] Type optimization

🐛 Bugfix

- [x] Solve the issue and close #1084 
- [x] 修复行头树形结构下对齐错误 

🔧 Chore

- [ ] Test case
- [x] 添加单元格自定义对齐方式文档
- [x] 减小自定义icon文档中icon的尺寸 
- [ ] CI / workflow
- [ ] Release version
- [ ] Other ()

### 📝 Description
梳理单元格对齐方式，修复行头对齐bug
image

### 🖼️ Screenshot

| Before | After |
| ------ | ----- |
| image   | image  |

### 🔍 Self Check before Merge

- [ ] Add or update relevant Docs.
- [ ] Add or update relevant Demos.
- [ ] Add or update relevant TypeScript definitions.

### CODE REVIEW

File: s2-site/docs/manual/advanced/custom/cell-align.zh.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+因此，单元格的对齐方式是有一定限制的。下面分别介绍每个类型单元格可以自定义的对齐行为。
+
+## 角头对齐
+
+* 行头单元格（红色部分）对齐方式受 [bolderText](/zh/docs/api/general/S2Theme#defaultcelltheme) 控制
+* 列头单元格（蓝色部分）对齐方式受 [text](/zh/docs/api/general/S2Theme#defaultcelltheme) 控制
+
+![img](https://gw.alipayobjects.com/zos/antfincdn/6wPCHImDZ/b36ca38e-aa8e-4ef6-a903-b9d605204de0.png)
+
+角头的`textBaseline`被内部规定为`middle`，因此只能自定义`textAlign`，下面是三种对齐方式的展示形态：
```
Comment (lijinke666) [inline s2-site/docs/manual/advanced/custom/cell-align.zh.md:23]: 单词和汉字间加空格

**Review by lijinke666 — APPROVED** [review]

File: packages/s2-core/src/theme/index.ts [hunk 2]
```diff
@@ -30,10 +30,10 @@ export const getTheme = (
       text: {
         fontFamily: FONT_FAMILY,
         fontSize: 12,
-        fontWeight: 'normal',
+        fontWeight: isWindows() ? 'bold' : 500,
         fill: basicColors[0],
         opacity: 1,
-        textAlign: isTable ? 'center' : 'left',
+        textAlign: 'right',
```
Comment (xingwanying) [inline packages/s2-core/src/theme/index.ts:36]: 这个对明细表会不会有影响哦，而且为什么变成靠右了？这里设计规范是靠左哦

Comment (wjgogogo) [reply packages/s2-core/src/theme/index.ts:36]: 明细表没有角头哦，这个是将角头中行头的标题和列头标题对齐方式区分开，之前只有bolderText生效，无法分开控制

**Pull request merged**